Dockerfile

```diff
@@ -14,8 +14,17 @@ LABEL org.opencontainers.image.source="https://github.com/interlynk-io/sbomqs"
 LABEL org.opencontainers.image.description="Quality metrics for your sboms"
 LABEL org.opencontainers.image.licenses=Apache-2.0
 
+# Copy the certs from the builder image
+COPY --from=builder /bin/bash /bin/bash
+COPY --from=builder /bin/grep /bin/grep
+COPY --from=builder /lib /lib
+COPY --from=builder /usr /usr
```

Comment:
`/usr` from `golang:1.22.2-alpine` increases the image size considerably about 71 MiB. Maybe copy the `/usr` folder from `alpine:3.19` or use `alpine:3.19` as a base image instead of `scratch`?

```
FROM alpine:3.19 AS alpine

FROM scratch
...
COPY --from=alpine /usr /usr
...
```